src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = 40 * days;
+
+  switch (true) {
+    case days >= 3 && days < 7: {
+      return rentalCost - 20;
+    }
+
+    case days >= 7: {
+      return rentalCost - 50;
+    }
+  }
```

Comment:
Using a switch statement for boolean conditions is unconventional and less readable than using if-else statements. Consider refactoring to use if-else for clarity.